wpk/run.sh

```diff
@@ -85,6 +85,7 @@ main() {
     echo "PACKED FILE -> $OUTPUT"
     # Update versions file
     cd ${OUTDIR}
+    sha512sum "${OUTPUT}" > "${OUTPUT}".sha512 
```

Comment:
This is the only script that will always compute the sha512 checksum of the package. I think that you should add the option as you did in the other scripts.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/192/commits/177abf51396ea9459086088f4a1d748bce6b703f